examples/lighting-app/lighting-common/lighting-app.zap

```diff
@@ -0,0 +1,683 @@
+{
+  "writeTime": "Thu Oct 29 2020 15:13:57 GMT-0400 (GMT-04:00)",
+  "featureLevel": 1,
+  "creator": "zap",
+  "keyValuePairs": [
+    {
+      "key": "commandDiscovery",
+      "value": "1"
+    },
+    {
+      "key": "defaultResponsePolicy",
+      "value": "always"
+    },
+    {
+      "key": "filePath",
+      "value": "/Users/jepenven/workspace/connectedhomeip/examples/lock-app/lock-common/lock_app.zap"
```

Comment:
These paths seem to be user-specific. Are they ignored? Any chance we can make them generic somehow?

Reply:
The path seems to refer to itself. I'm not sure what is the uses for this key/value pair.

Adding @jepenven-silabs to understand what is the use of it.

Reply:
It's a known issue with ZAP. Should be fixed by @tecimovic soon

Reply:
We have on our plate to fix this shortly.
Appbuilder solution, which we plan to replicate, without having a different idea, was to store all paths in "relative to X" mode, where X could be:
   a.) relative to the ZAP file itself.
   b.) relative to the root of SDK (which is passed in as argument).
   c.) relative to the "project path", where IDE is responsible to pass in what "project path" is.
   d.) relative to the install location of zap
   e.) relative to the home directory of the user.
   f.) fully absolute.

User chooses in the UI what is the "relativity mode" for these paths, as they are stored in the zap file, and tooling enforces the existence of the proper "relativity point" in cases where it's required.

This worked for wide range of possibilities for collaborating teams which might decide how to standardize on stuff differently.

If anyone has a different good idea around this, I'll be all ears....

Reply:
> We have on our plate to fix this shortly.
> Appbuilder solution, which we plan to replicate, without having a different idea, was to store all paths in "relative to X" mode, where X could be:
> a.) relative to the ZAP file itself.
> b.) relative to the root of SDK (which is passed in as argument).
> c.) relative to the "project path", where IDE is responsible to pass in what "project path" is.
> d.) relative to the install location of zap
> e.) relative to the home directory of the user.
> f.) fully absolute.

Just curious. In the ZAP use case what is the difference between b.) and d.) currently ?

In our case a.) seems to fit well if I understand correctly.